src/WebJobs.Script.WebHost/Middleware/SystemTraceMiddleware.cs

```diff
@@ -42,15 +42,15 @@ public async Task Invoke(HttpContext context)
             {
                 [ScriptConstants.LogPropertyActivityIdKey] = request.GetRequestId()
             };
-            _logger.Log(LogLevel.Information, 0, logData, null, (s, e) => $"Executing HTTP request: {details}");
+            _logger.Log(LogLevel.Debug, 0, logData, null, (s, e) => $"Executing HTTP request: {details}");
 
             await _next.Invoke(context);
 
             sw.Stop();
             details["identities"] = GetIdentities(context);
             details["status"] = context.Response.StatusCode;
             details["duration"] = sw.ElapsedMilliseconds;
-            _logger.Log(LogLevel.Information, 0, logData, null, (s, e) => $"Executed HTTP request: {details}");
+            _logger.Log(LogLevel.Debug, 0, logData, null, (s, e) => $"Executed HTTP request: {details}");
         }
```

Comment:
Note that I adjusted these down to Debug as they're very verbose and will now appear in App Insights / file / etc logs.

@pragnagopa we may want to do something similar with the LanguageWorker startup logs -- just try running this against a node/python app and you can now see a lot more logs than you could previously. We can adjust that after this goes in, though.